CONTRIBUTING.md

```diff
@@ -15,24 +14,24 @@ inclusion.
 ## Pull Request Process
 
 1. Features should generally be pulled against the `master` branch.
-   Bug-fixes, depending on the urgency and risk, goes either against an
-   active release-branch, or `master`.
+   Bug-fixes, depending on the urgency and risk, go either against an active
```

Comment:
Should be "Bug fixes" to match the change further down.

Reply:
Yeah, I'll fix it :)